SwiftShell/Print.swift

```diff
@@ -11,7 +11,7 @@ import Foundation
 func print(command: String, with arguments: [String]? = nil) {
     var output = "SwiftShell $ \(command)"
     if let arguments = arguments {
-        output += arguments.reduce("", { $0 + " " + $1 })
+        output += arguments.reduce("", combine: { $0 + " " + $1 })
```

Comment:
Revert this change. You should update to Xcode 8 Beta 4. Swift 3 beta 4 removed combine label.